lib/cli/daemoncommand.cpp

```diff
@@ -298,6 +330,8 @@ int RunWorker(const std::vector<std::string>& configs, bool closeConsoleLog = fa
 
 	ApiListener::UpdateObjectAuthority();
 
+	NotifyStatus("Activated all objects.");
```

Comment:
This is the message displayed while Icinga 2 is running normally, isn't it? I think it would make sense to use something that better communicates this. Maybe something like "Startup finished", "Running normally" or possibly something nice like "Taking care of your monitoring"?